src/components/05_pages/NodeEditForm/index.js

```diff
@@ -77,12 +81,34 @@ styles = {
   `,
 };
 
+const extractRestorableEntity = (state, entity) => {
+  const restorableEntity = state.content.restorableContentEditById[entity.id];
+  if (
+    // @todo maybe that's too defensive?
+    restorableEntity &&
+    restorableEntity.data &&
+    restorableEntity.data.attributes &&
+    entity &&
+    entity.attributes &&
+    // Just offer to restoring when the loaded entity wasn't changed in the meantime.
```

Comment:
Maybe `Restoring content is only offered when the loaded entity wasn't changed in the meantime.` would read a bit better.

Reply:
That's indeed true. Thank you!